account_payment_partner/views/account_move_view.xml

```diff
@@ -35,20 +35,29 @@
                 <field name="payment_mode_filter_type_domain" invisible="1" />
                 <field name="partner_bank_filter_type_domain" invisible="1" />
             </field>
-            <field name="partner_bank_id" position="attributes">
+            <xpath expr="//group[@id='header_right_group']//field[@name='partner_bank_id']" position="attributes">
+                <attribute name="context">
+                    {'default_partner_id':commercial_partner_id}
+                </attribute>
                 <attribute name="domain">
                     [('partner_id', '=', partner_bank_filter_type_domain),
                     '|',('company_id', '=', company_id),('company_id', '=', False)]
                 </attribute>
                 <attribute name="attrs">
                     {'required': [('bank_account_required', '=', True),('move_type', 'in', ('in_invoice', 'in_refund'))],
                     'readonly': [('state', '!=', 'draft')],
-                    'invisible': [('move_type', '=', 'entry')]}
+                    'invisible': [('move_type', 'not in', ('in_invoice', 'in_refund', 'in_receipt'))]}
```

Comment:
It should be shown on customer invoices as well

Reply:
The problem is that without the change, the partner_bank_id field that is being modified in the xml is the one shown in supplier invoices, causing it to be shown in both supplier invoices and customer invoices and this is incorrect.

For this reason it is necessary to apply the attribute changes to the two partner_bank_id fields in the base odoo xml. 

- https://github.com/OCA/bank-payment/pull/1072/files#diff-eb13f3e50e164744b9ed2075e137dc0d7dc5ea097a6fa128d7c2366a0241f432R38
- https://github.com/OCA/bank-payment/pull/1072/files#diff-eb13f3e50e164744b9ed2075e137dc0d7dc5ea097a6fa128d7c2366a0241f432R52

With the change, now only the correct field is shown on customer invoices.

- Customer Invoice:

![Selección_321](https://user-images.githubusercontent.com/6359121/231791080-77051216-bc53-400a-8d1a-70de47360ed7.jpg)

- Bills:

![Selección_323](https://user-images.githubusercontent.com/6359121/231791381-e8abb7af-f11f-4d97-85f8-19aaf096da32.jpg)

Regards